src/renderer/component/header/view.jsx

```diff
@@ -38,68 +38,67 @@ const Header = (props: Props) => {
     <header className="header">
       <div className="header__navigation">
         <Button
-          noPadding
-          button="alt"
-          icon={icons.HOME}
-          className="btn--home-nav"
-          description={__('Home')}
-          onClick={() => navigate('/discover')}
-        />
-        <div className="header__history">
-          <Button
-            className="btn--arrow"
-            icon={icons.ARROW_LEFT}
-            description={__('Navigate back')}
-            onClick={back}
-            disabled={isBackDisabled}
-          />
-          <Button
-            className="btn--arrow"
-            icon={icons.ARROW_RIGHT}
-            description={__('Navigate forward')}
-            onClick={forward}
-            disabled={isForwardDisabled}
-          />
-        </div>
-      </div>
-      <WunderBar />
-      <div className="header__actions-right">
-        <Button
-          button="inverse"
-          className="btn--header-balance"
-          onClick={() => navigate('/wallet')}
+          className="header__navigation-item header__navigation-item--wallet"
+          description={__('Your wallet')}
+          iconRight="LBC"
           label={
             isUpgradeAvailable ? (
               `${balance}`
             ) : (
               <React.Fragment>
-                <span className="btn__label--balance" title={`${balance} LBC`}>
-                  You have
-                </span>{' '}
-                <span title={`${balance} LBC`}>{roundedBalance} LBC</span>
+                <span title={`${balance} LBC`}>{roundedBalance}</span>
               </React.Fragment>
             )
           }
-          iconRight="LBC"
-          description={__('Your wallet')}
+          onClick={() => navigate('/wallet')}
         />
 
         <Button
-          uppercase
-          button="primary"
-          className="btn--header-publish"
-          onClick={() => navigate('/publish')}
-          icon={icons.UPLOAD}
```

Comment:
With the new icons that were added we now have two ways of adding icons to buttons. We should consolidate this into one.

Reply:
What is being consolidated?

Reply:
We should either always use a class and add the icons through css with `background-image` like you do [here](https://github.com/lbryio/lbry-desktop/pull/2102/files#diff-88b5fbd05c1ee805da8402786b5cc714R58), or stick with using props and explicitly render an icon like [we are currently doing](https://github.com/lbryio/lbry-desktop/blob/master/src/renderer/component/common/icon.jsx#L3) where we use `react-feather`

Reply:
What happens when we want to use an icon that doesn't exist with `react-feather`?

Reply:
In the past, we have just chosen a different icon. This comment was more about still using the `Icon` component. I think it's ok if we use different icons there (it should be avoided, but fine if absolutely necessary), but we definitely shouldn't be adding icons in two different ways.

Everything should go through the `Icon` component and we shouldn't be adding icons by using a class (`.play`). I have some ideas but maybe we should discuss over slack. We could check to see if we have a custom icon, if one isn't there, then fallback to react-feather.